lib/start-service-auth-result.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { TrustServiceAuthParameters } = require('./trust-service-auth-parameters.js');
+
+class StartServiceAuthResult {
```

Comment:
Please, add the comments following the same patterns I mentioned in GetServiceAuthResult